controllers/flinkcluster_submit_job_script.go

```diff
@@ -31,78 +31,101 @@ var submitJobScript = `
 
 set -euo pipefail
 
-JOB_MANAGER="$2"
-
-function list_jobs() {
-	for i in {1..10}; do
-		if /opt/flink/bin/flink list -a --jobmanager "${JOB_MANAGER}" 2>&1; then
-			return 0
-		else
-			sleep 5
-		fi
-	done
-
-	echo "Failed to list jobs." >&2
-	return 1
+readonly TERM_LOG="/dev/termination-log"
```

Comment:
Add a comment for this script, especially how the termination log will look like when submission fails or succeeds.

Reply:
Commented.